src/transformers/models/swin/configuration_swin.py

```diff
@@ -145,3 +151,20 @@ def __init__(
         # we set the hidden_size attribute in order to make Swin work with VisionEncoderDecoderModel
         # this indicates the channel dimension after the last stage of the model
         self.hidden_size = int(embed_dim * 2 ** (len(depths) - 1))
+
+
+class SwinOnnxConfig(OnnxConfig):
+
+    torch_onnx_minimum_version = version.parse("1.11")
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("pixel_values", {0: "batch", 1: "sequence"}),
```

Comment:
We recently refactored the dynamic axis name for image models - please replace with the following:

```suggestion
                ("pixel_values", {0: "batch", 1: "num_channels"}),
```